.github/workflows/ci-check-bundle-size.yml

```diff
@@ -0,0 +1,88 @@
+name: Check Binaries bundle size
```

Comment:
this is checking binary size, nothing to do with "bundle" (bundle happens in the UI)